models/ens/ens_ethereum_schema.yml

```
@@ -305,4 +305,30 @@ models:
               values: ['Queued', 'Active', 'Executed', 'Canceled','Defeated']
       - &description
         name: description
-        description: "Description of the proposal"
\ No newline at end of file
+        description: "Description of the proposal"
+
+  - name: ens_set_name_latest
+    meta:
+      blockchain: ethereum
+      project: ethereum_name_service
+      contributors: sankinyue
+    config:
+      tags: ['ethereum','ens','ethereum_name_service','ethereumnameservice','setname','latest']
+    description: "View the latest ENS set name records"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - address
+            - name
+            - last_tx_hash
```

We try the less combination first.
```suggestion
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - address
            - name
```